stratum/docs/setup_guide_barefoot_tofino_onos.md

```diff
@@ -0,0 +1,191 @@
+# Installation guide:<br/>ONOS + Stratum + Barefoot Tofino switch
```

Comment:
License header

Reply:
Added
```
<!--
Copyright 2020-present Open Networking Foundation

SPDX-License-Identifier: Apache-2.0
-->
```
Is this right?

Reply:
Close, just the copyright holder. You wrote this guide, so it's either you or your employer.

Reply:
Like so?
```
<!--
Copyright 2020 Technische Universität Darmstadt
Copyright 2020-present Open Networking Foundation

SPDX-License-Identifier: Apache-2.0
-->
```